contributing.md

```diff
@@ -0,0 +1,21 @@
+---
+---
+
+# Contributing to How We Have Been Saving The Planet This Week
+
+You can help out by suggesting stories and subjects to [helpdesk@newatoms.com](mailto:helpdesk@newatoms.com). The newsletter and it's site are open source, you can contribute directly on [our GitHub repository](https://github.com/newatoms/hwhbstptw).
```

Comment:
Can contributors also merge or only do pull requests? I think the latter is favourable, no?